renku/core/management/workflow/plan_factory.py

```diff
@@ -466,10 +525,11 @@ def add_command_output_from_input(self, input: CommandInput, name):
     def add_command_output_from_parameter(self, parameter: CommandParameter, name):
         """Create a CommandOutput from a parameter."""
         self.parameters.remove(parameter)
-        value = Path(self._path_relative_to_root(parameter.default_value))
+        value = Path(self._path_relative_to_root(parameter.actual_value))
```

Comment:
the self._path_relative_to_root, does actually a `(self.directory / path).resolve()` to the input path, so the only thing that might be worth to consider is to actually use there `get_relative_path` instead of pathlib's function... but i'm not so sure if it's worth it...